avae/utils.py

```diff
@@ -50,6 +59,263 @@ def dims_after_pooling(start: int, n_pools: int) -> int:
     return start // (2**n_pools)
 
 
+def pass_batch(
```

Comment:
Let's move this into a separate file specifically to do with NN training

Reply:
done!